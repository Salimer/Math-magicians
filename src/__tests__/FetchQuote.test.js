import { render } from '@testing-library/react';
import FetchQuote from '../components/FetchQuote';

test('renders FetchQuote correctly', () => {
  const { conqatiner } = render(
    <FetchQuote />,
  );
  expect(conqatiner).toMatchSnapshot();
});
