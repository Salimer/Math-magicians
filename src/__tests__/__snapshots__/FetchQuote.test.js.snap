// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FetchQuote correctly 1`] = `undefined`;
